meta-oe/recipes-connectivity/hiba/hiba_git.bb

```diff
@@ -0,0 +1,50 @@
+SUMMARY = "Google Host Identity Based Authorization"
+DESCRIPTION = "HIBA is a system built on top of regular OpenSSH \
+  certificate-based authentication that allows to manage flexible \
+  authorization of principals on pools of target hosts without the need to \
+  push customized authorized_users files periodically."
+
+S = "${WORKDIR}/git/hiba"
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://${S}/LICENSE;md5=7e161abe9a4585310725dd86c28d1ae2"
+
+PV = "git${SRCPV}"
+
+inherit autotools
+
+DEPENDS = "openssl openssh zlib"
+RDEPENDS:${PN} = " bash"
+
+SRC_URI = " \
+  git://github.com/google/hiba;protocol=https;branch=main;destsuffix=git/hiba;name=hiba \
+  git://github.com/openssh/openssh-portable.git;protocol=https;branch=master;destsuffix=git/openssh-portable;name=openssh \
+"
+SRCREV_hiba = "cae606dafc5692240ac49441fd57653d7aade99b"
```

Comment:
I added some changes to HIBA recently.

The new head is at 3a6beb475ebd4dff3223327039738b30240f0c43